modules/effects/arrow-effects/src/main/kotlin/arrow/effects/internal/Utils.kt

```diff
@@ -74,6 +74,19 @@ object Platform {
       is Either.Right -> Some(eitherRef.b)
     }
   }
+
+  /**
+   * Composes multiple errors together, meant for those cases in which error suppression, due to a second error being
+   * triggered, is not acceptable.
+   *
+   * On top of the JVM this function uses Throwable#addSuppressed, available since Java 7. On top of JavaScript the
```

Comment:
Does this mean it may not work on older versions of Android? Or was this pat of the "Java 6.5" update too?

Reply:
https://developer.android.com/reference/java/lang/Throwable#addSuppressed(java.lang.Throwable)

Minimum API 19. What's our expected level of support these days?

Reply:
Good question, I completely missed this when working on it.

Since `minSdk` tweeted in Nov 2016 that they recommend min API 19, I'd think we're good. [source](https://twitter.com/minSdkVersion/status/795702388774567936). But perhaps we should ask around? I'll create a ticket for this.